single-servizio.php

```diff
@@ -405,7 +405,7 @@ function convertToPlain($text) {
 
                             <section class="it-page-section">
                                 <h2 class="mb-3" id="contacts">Contatti</h2>
-                                <div class="row">
+                                <div class="row" data-element="service-area">
```

Comment:
Forse non nella `div.row`, che qui è in comune con la sez. Argomenti.
Probabilmente il _data-element_ andrebbe avvicinato alla card UO, in modo da riferirsi solo a quella.

Reply:
@enrimk forse sarebbe meglio applicare comunque il `data-element` al `<div>` con classe `.card` che agisce da vero elemento genitore

Reply:
ok, va bene @enrimk